ui/apps/platform/src/routePaths.js

```diff
@@ -145,12 +152,18 @@ const vulnManagementPathToLabelMap = {
     [vulnManagementRiskAcceptancePath]: 'Risk Acceptance',
 };
 
+const vulnerabilitiesPathToLabelMap = {
+    [vulnerabilitiesBasePath]: 'Vulnerabilities',
+    [vulnerabilitiesWorkloadCvesPath]: 'Workload CVEs',
+};
+
 export const basePathToLabelMap = {
     [dashboardPath]: 'Dashboard',
     [networkBasePath]: 'Network Graph (1.0)',
     [networkBasePathPF]: 'Network Graph (2.0 preview)',
     [violationsBasePath]: 'Violations',
     [complianceBasePath]: 'Compliance',
+    ...vulnerabilitiesPathToLabelMap,
```

Comment:
If container pages take responsibility to render `PageTitle` element, can we stop this invisible global connection?